packages/terra-clinical-onset-picker/tests/nightwatch/component/LessMonth.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Base from 'terra-base';
+import moment from 'moment';
+import OnsetPicker from '../../../lib/OnsetPicker';
+
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+
+export default () =>
+  <Base locale={locale}>
+    <OnsetPicker
```

Comment:
Add some context to these tests. A `<p>` with a description of what should be displayed and the functionality that's being tested.